arithmetization/src/main/java/net/consensys/linea/zktracer/module/ecdata/EcData.java

```diff
@@ -97,17 +87,38 @@ public void commit(List<MappedByteBuffer> buffers) {
     int stamp = 0;
     long previousId = 0;
 
+    // TODO: operations should come in the right order now, no need to sort them. To test.
     List<EcDataOperation> sortedOperations =
         this.operations.stream().sorted(Comparator.comparingLong(EcDataOperation::id)).toList();
 
     for (EcDataOperation op : sortedOperations) {
       // TODO: temporary hack, we should have only successful EcData Operations, will be fixed in PR
-      // #748
+      //  #748
       if (op.successBit()) {
```

Comment:
@lorenzogentile404 @OlivierBBB  could you check this plz ? 
I think that now, when we call EcData, we trace it. 
Could be problematic to keep it like this, as for ecRecover, we have `successBit = !recoveredAddress.isZero();`